divvunspell-bin/src/main.rs

```diff
@@ -95,12 +95,10 @@ fn run(
     suggest_cfg: &SpellerConfig,
 ) {
     for word in words {
-        let is_correct = speller.clone().is_correct_with_config(&word, &suggest_cfg);
-        writer.write_correction(&word, is_correct);
-
-        if is_suggesting && (is_always_suggesting || !is_correct) {
-            let suggestions = speller.clone().suggest_with_config(&word, &suggest_cfg);
-            writer.write_suggestions(&word, &suggestions);
+        let cleaned_str = Tokenize::word_indices(word.as_str());
```

Comment:
Using a trait this was is valid, but the more idiomatic way is to import `Tokenize` at the top of the file and then do `word.as_str().word_indices()`